ukvm/ukvm_elf.c

```diff
@@ -128,7 +128,9 @@ void ukvm_elf_load(const char *file, uint8_t *mem, size_t mem_size,
             || hdr.e_ident[EI_MAG3] != ELFMAG3
             || hdr.e_ident[EI_CLASS] != ELFCLASS64
             || hdr.e_type != ET_EXEC
-#if defined(__x86_64__)
+#if defined(__OpenBSD__) && defined(__x86_64__)
+            || hdr.e_machine != EM_AMD64
```

Comment:
Presumably this is the same value as `EM_X86_64` but OpenBSD calls it `EM_AMD64`?

Reply:
just a quick look on the net, both are 62.

[OpenBSD definition](https://github.com/openbsd/src/blob/master/sys/sys/exec_elf.h#L196)
[Linux definition](https://github.com/torvalds/linux/blob/ead751507de86d90fa250431e9990a8b881f713c/include/uapi/linux/elf-em.h#L31)

Reply:
When i first wrote the code you presumption was correct, i went back and checked when i had more time and it does defined both, so i switched back to `EM_X86_64`

Reply:
Thanks!